tests/performance-test/Dockerfile

```diff
@@ -14,7 +14,9 @@ FROM tripleomaster/centos-binary-collectd:current-tripleo-rdo
 USER root
 
 RUN yum install golang -y && \
-    yum update -y && \
+    #yum update -y && \  # skipping this step due to issue with
+                         # centos-binary-collectd image. More info:
+                         # https://github.com/redhat-service-assurance/telemetry-framework/issues/81
```

Comment:
Thanks for this. I'd have been okay with just this line. :) I see the bug even has the error message, so bonus points!